### PR TITLE
Update Travis.yml

### PR DESCRIPTION
**Pull request by shinnn**
* Introduced [container-based infastructure](http://docs.travis-ci.com/user/workers/container-based-infrastructure/) for faster build
* Added `node` (latest Node) and `iojs` (latest io.js) to the version list
* Removed `0.8` and `0.11` from the list
* Removed `script` field since Travis CI runs `npm test` by default

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,15 @@
+sudo: false
 language: node_js
 before_install:
   - curl --location http://git.io/1OcIZA | bash -s
```
Comment (rvagg) [inline .travis.yml]: I'm not happy including this, the control of this gist is outside of the control of this repo and it's a bit too opaque, I'd rather either leave this out or put the logic locally

Comment (shinnn) [inline .travis.yml]: You're talking about [L4](https://github.com/rvagg/bl/pull/21#discussion-diff-32892797R4)? That's not related to this PR but it's OK to remove it.

Comment (shinnn) [inline .travis.yml]: Basically we can remove the `before_install` script if we remove [this line](https://github.com/shinnn/bl/blob/travis-yml/.travis.yml#L6). I think we don't need to test the module with Node v0.8 that is too outdated. Actually, [your other project dropped it](https://github.com/rvagg/through2/commit/0fa42c010aa58b562fd3eabfb53c0bc5ddaa60f7).

Comment (rvagg) [inline .travis.yml]: yep, happy to drop 0.8

Comment (shinnn) [inline .travis.yml]: https://github.com/rvagg/bl/commit/ba0fad3640ce1d01359cb5d42ad70e60692a793e

**Comment by rvagg** [comment]
merged! thanks for this, I haven't been keeping up with travis developments for a while and missed the whole container-based stuff

**Comment by shinnn** [comment]
You're welcome!

**Pull request closed without merging**